projects/baby_driver/scripts/can_message.py

```diff
@@ -0,0 +1,20 @@
+""" This module sends a CAN message for babydriver"""
+
+import can
+
+def send_message(message):
+    """sends can message with 8 bytes. You can change the 2nd one"""
+    # change the channel later to can0 or vcan0
+    bus = can.interface.Bus(bustype='socketcan', channel='can0', bitrate=500000)
+
+    msg = can.Message(
+        arbitration_id=0x2,
```

Comment:
It looks like I was wrong, `arbitration_id` is 11 bits: 4 bits for the source ID, 1 bit for type (0=data, 1=ack), 6 bits for the message ID. `send_message` should probably take this into account or at least allow setting the arbitration ID.